src/ident/Ident.h

```diff
@@ -9,17 +9,24 @@
 #ifndef SQUID_SRC_IDENT_IDENT_H
 #define SQUID_SRC_IDENT_IDENT_H
 
-#if USE_IDENT
-
 #include "cbdata.h"
 #include "comm/forward.h"
+#include "sbuf/forward.h"
+
+#include <optional>
 
 typedef void IDCB(const char *ident, void *data);
 
 /// Ident Lookup API
 namespace Ident
 {
 
+/// rfc931 user identity
+/// An existing value indicates a successfully received response
+/// from Ident server: empty string means that USERID response part
+/// was missing.
```

Comment:
AFAICT, the proposed description does not match PR IdentLookup::LookupDone() code because that code stores a dash (instead of an empty value) when IDENT lookup fails. Please polish the description and adjust LookupDone() to match (i.e. to store an empty value instead of dash_str).

```suggestion
/// A user-id field of an RFC 931 auth-reply response.
/// Value existence implies that there was an attempt to obtain a valid user-id.
/// A non-empty value implies that the above attempt was successful.
```